probcal/utils/experiment_utils.py

```diff
@@ -74,6 +78,14 @@ def get_datamodule(
                 num_workers=num_workers,
                 persistent_workers=True if num_workers > 0 else False,
             )
+        elif dataset_spec == ImageDatasetName.AAF:
+            return AAFDataModule(
+                csv_file_path = os.path.join(GLOBAL_DATA_DIR, "aaf/image_sets/picture_data.csv"),
+                root_dir=os.path.join(GLOBAL_DATA_DIR, "aaf/original_images"),
```

Comment:
As things currently stand, the CSV file path doesn't exist in other people's copies of the repo. And `root_dir` also doesn't exist by default